media-gfx/blender/blender-9999.ebuild

```diff
@@ -90,7 +90,7 @@ RDEPEND="${PYTHON_DEPS}
 	)
 	opensubdiv? ( >=media-libs/opensubdiv-3.4.0[cuda=] )
 	openvdb? (
-		>=media-gfx/openvdb-8.2.0-r2:=
+		>=media-gfx/openvdb-8.2.0-r2:=[nanovdb?]
```

Comment:
Seeing that you only added the nanovdb USE flag to v9.0.0, shouldn't we restrict this to >=openvdb-9.x?

Reply:
Yeah, I think I'll just be lazy with this and bump support to 9.x as you suggested.
Cleaner that way.